tutor/templates/build/openedx/Dockerfile

```diff
@@ -181,8 +181,14 @@ COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
 
 # Pull latest translations via atlas
-RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --branch='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} \
-    translations/edx-platform/conf/locale:conf/locale
+RUN ./manage.py lms --settings=tutor.i18n pull_plugin_translations --verbose --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }}
+RUN ./manage.py lms --settings=tutor.i18n pull_xblock_translations --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }}
+RUN atlas pull --repository='{{ ATLAS_REPOSITORY }}' --revision='{{ ATLAS_REVISION }}' {{ ATLAS_OPTIONS }} \
+    translations/edx-platform/conf/locale:conf/locale \
+    translations/studio-frontend/src/i18n/messages:conf/plugins-locale/studio-frontend
+RUN ./manage.py lms compile_xblock_translations
+RUN ./manage.py cms compile_xblock_translations
```

Comment:
Why not use the `--settings=tutor.i18n` option with the `compile_xblock_translations` commands? As far as I know, loading production settings just fails during build.

(I wish I could test this but I don't have great internet connection right now)

Reply:
100%. Good catch. I thought this change has already been made. Now it's done.

`$ tutor images build --no-cache openedx` works locally for me. We'll see if it breaks for someone else.

Reply:
Now it builds correctly:

```
 => [production 16/33] COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/                                                                            0.1s
 => [production 17/33] COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/                                                                            0.0s
 => [production 18/33] RUN ./manage.py lms --settings=tutor.i18n pull_plugin_translations --verbose --repository='openedx/openedx-translations' --revisio  6.7s
 => [production 19/33] RUN ./manage.py lms --settings=tutor.i18n pull_xblock_translations --repository='openedx/openedx-translations' --revision='main'    6.0s
 => [production 20/33] RUN atlas pull --repository='openedx/openedx-translations' --revision='main'      translations/edx-platform/conf/locale:conf/local  3.7s
 => [production 21/33] RUN ./manage.py lms --settings=tutor.i18n compile_xblock_translations                                                               4.5s
 => [production 22/33] RUN ./manage.py cms --settings=tutor.i18n compile_xblock_translations                                                               4.7s
 => [production 23/33] RUN ./manage.py lms --settings=tutor.i18n compile_plugin_translations                                                               4.0s
 => [production 24/33] RUN ./manage.py lms --settings=tutor.i18n compilemessages -v1                                                                       3.2s
 => [production 25/33] RUN ./manage.py lms --settings=tutor.i18n compilejsi18n                                                                             3.8s
 => [production 26/33] RUN ./manage.py cms --settings=tutor.i18n compilejsi18n     
```